assets/js/modules/analytics/components/common/ExistingGTMPropertyNotice.js

```diff
@@ -38,17 +38,17 @@ export default function ExistingGTMPropertyNotice() {
 		select( MODULES_TAGMANAGER ).getSingleAnalyticsPropertyID()
 	);
 
-	if ( ! gtmAnalyticsPropertyID ) {
-		return null;
-	}
+	// if ( ! gtmAnalyticsPropertyID ) {
+	// 	return null;
+	// }
```

Comment:
I think this was left out by mistake.

```suggestion
	if ( ! gtmAnalyticsPropertyID ) {
		return null;
	}
```